opal-core-ws/src/main/java/org/obiba/opal/web/vcf/VCFStoreResourceImpl.java

```diff
@@ -66,6 +67,13 @@ public void setVCFStore(String serviceName, String name) {
   }
 
   @Override
+  public Response delete() {
+    removeSamplesMappings();
+    store.getVCFNames().forEach(this::deleteVCF);
```

Comment:
Call VCFStoreService.deleteStore() instead as it does more than simply deleting the VCFs